lib/gds_api/test_helpers/business_support_helper.rb

```diff
@@ -15,28 +15,27 @@ def setup_business_support_stubs(endpoint, path)
           else
             results = @stubbed_business_supports
           end
-          {:body => plural_response_base.merge("results" => results, "total" => results.size).to_json}
+          { body: plural_response_base.merge("results" => results, "total" => results.size).to_json }
         end
-
       end
 
-      def api_has_business_support(business_support, facets={})
+      def api_has_business_support(business_support, facets = {})
         facets = sanitise_facets(facets)
-        if business_support.is_a?(Symbol)
-          bs_with_facets = facets.merge(:title => business_support)
-        else
-          bs_with_facets = facets.merge(business_support)
-        end
+        bs_with_facets = if business_support.is_a?(Symbol)
```

Comment:
This isn't that cop actually. I get the following report:

```
lib/gds_api/test_helpers/business_support_helper.rb:25:11: C: Style/IndentationWidth: Use 2 (not -15) spaces for indentation.  (https://github.com/bbatsov/ruby-style-guide#spaces-indentation)
          facets.merge(title: business_support)
          ^^^^^^^^^^^^^^^
```

I think the `if`, `else` and `end` should be aligned. I'll check if I can make the linter pass differently.